rpc-client-api/src/client_error.rs

```diff
@@ -0,0 +1,199 @@
+pub use reqwest;
+use {
+    crate::{rpc_request, rpc_response},
+    solana_faucet::faucet::FaucetError,
+    solana_sdk::{
+        signature::SignerError, transaction::TransactionError, transport::TransportError,
+    },
+    std::io,
+    thiserror::Error,
+}; // export `reqwest` for clients
```

Comment:
As I mentioned above, for now, this export is still happening. Do you want the comment removed?

Reply:
heh oh I missed your previous comment. I'm guessing that `cargo fmt` puts this comment in this disembodied place? I was just suggesting that it be removed entirely if it can't be placed above the actual export anymore

Reply:
Hehe, got it! Thanks for clarifying :)